bigtable/garbagecollection/garbagecollection_test.go

```diff
@@ -54,6 +55,16 @@ func TestGarbageCollection(t *testing.T) {
 		t.Errorf("got %q, want %q", got, want)
 	}
 
+	buf.Reset()
+	if err = updateGCRule(buf, project, instance, tableName); err != nil {
+		t.Errorf("TestGarbageCollection: %v", err)
+	}
+
+	got = buf.String()
+	if want := "Updated column family cf1 GC rule with policy: versions() > 1\n"; !strings.Contains(got, want) {
```

Comment:
This looks like the full string? We usually like to check a smaller amount to avoid breaking the test on small changes.

Reply:
I don't really have an issue with that. If people change the code, they should change the test too.